docs/docs/03-language-guide/90-reference.md

```diff
@@ -229,71 +213,50 @@ let jsonMutObj = MutJson {
 
 The `Json` keyword can be omitted from `Json` object literals:
 
-```js
+```TS
 let jsonObj = { boom: 123, bam: [4, 5, 6] };
 ```
 
 Every value within a `Json` array or object also has a type of `Json`.
 
 ##### 1.1.4.2 JSON objects
 
-To access a field within an object, use the `.` notation:
+To access a field within an object, use `.get("{field name}")`:
 
-```js
-let boom: Json = jsonObj.boom;
-```
-
-Trying to access a non-existent field will fail at runtime. For example:
-
-```js
-log(jsonObj.boom.dude.world);
-// RUNTIME ERROR: Uncaught TypeError: Cannot read properties of undefined (reading 'world')
+```TS
+let boom: Json = jsonObj.get("boom");
 ```
 
-Like in JavaScript, it is also possible to access object fields using `[]`:
+Trying to access a non-existent field will fail at compile time. For example:
```

Comment:
I think it would have to fail at runtime, since we can't always know ahead of time what value or fields are contained in a `Json` variable.
```suggestion
Trying to access a non-existent field will fail at runtime. For example:
```